roles/kubernetes-apps/csi_driver/vsphere/defaults/main.yml

```diff
@@ -13,7 +13,6 @@ vsphere_csi_snapshotter_image_tag: "v5.0.1"
 vsphere_csi_node_driver_registrar_image_tag: "v2.5.0"
 vsphere_csi_driver_image_tag: "v2.5.1"
 vsphere_csi_resizer_tag: "v1.4.0"
-
```

Comment:
Not really agree with removing the line break here, the csi_namespace is separated from the csi component versions